src/components/DonateButtonModal.jsx

```diff
@@ -350,10 +350,20 @@ const DonateButtonModal = props => {
     adminId,
     _amount,
     donationOwnerAddress,
+    community,
```

Comment:
Let's remove `community` and replace `adminId` with a structure holding more complete data of receiver project

Reply:
@aminlatifi 
It changed as you said

Reply:
@mohammadranjbarz Thanks